WebRoot/js/zimbraMail/calendar/view/ZmCalColView.js

```diff
@@ -2324,8 +2333,8 @@ function(data) {
 
         // Adjust apptOffset.x to be the offset from the clicked on column.  Then create the
         // start snap using this offset (so that start column of a multi-day is tracked).
-        var leftSnap = this._snapXY(data.apptX, data.apptY, 15);
-        var colSnap  = this._snapXY(data.apptX + data.apptOffset.x, data.apptY, 15);
+        var leftSnap = this._snapXY(data.apptX, data.apptY, ZmCalColView._SNAP_MINUTES);
+        var colSnap  = this._snapXY(data.apptX + data.apptOffset.x, data.apptY, ZmCalColView._SNAP_MINUTES);
```

Comment:
Question: the third parameter of `this._snapXY` was originally `15`. `ZmCalColView._SNAP_MINUTES` is `15` or `30`.  Is it okay not to set it to `ZmCalColView._SNAP_MINUTES/2` ?
Lines 2361, 2441 and 2450 as well.

Before the fix, the grid is 30 minutes slot, but an appointment can be moved in 15min unit by drag&drop.
After the fix, when `zimbraPrefCalenderScaling` is `30`, an appointment cannot be moved in 15 min but 30 min unit.

Reply:
Fixed. Keeping it to 15 in both the scenario to keep 30 minutes  experience as it is.